scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -257,9 +261,11 @@ internal fun replaceLicenseKeys(licenseExpression: String, replacements: Collect
 private fun getCopyrightFindings(result: JsonNode): List<CopyrightFinding> {
     val copyrightFindings = mutableListOf<CopyrightFinding>()
 
+    val input = getInputPath(result)
```

Comment:
how about naming this `inputPath` instead of just `input`?

Reply:
I'd like to stick to the same name as in `getLicenseFindings` (where `input` was already used before; the variable is named after the field name in the raw scan result).